webpack.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    entry: ["./public/scripts/history.js", "./public/scripts/confetti.js", "./public/scripts/main.js","./public/scripts/keymaster.js" ],
```

Comment:
Since main requires the others, do we need to spell out the rest? Would just including main work?